migration-guides/11.0.md

```diff
@@ -0,0 +1,10 @@
+# Major 11.0 Migration guide
```

Comment:
Do we allow this kind of breaking in v11 where we should only allow dependency breaking changes?

Reply:
We discussed about this change and agreed that bringing this breaking change on the major would bring more benefit than waiting to v12 to allow multi tag